autoload/jedi.vim

```diff
@@ -190,6 +190,9 @@ function! jedi#_vim_exceptions(str, is_eval)
     try
         if a:is_eval
             let l:result.result = eval(a:str)
+            if type(l:result.result) != type('')
+                let l:result.result = string(l:result.result)
+            endif
         else
```

Comment:
You decide. I find it really hard to judge. I think if it's a bug in neovim we should at least document it somehow in our sources, so that it's a little bit easier to follow why some things entered our codebase.